easybuild/easyblocks/c/cp2k.py

```diff
@@ -781,6 +782,19 @@ def install_step(self):
         except OSError, err:
             raise EasyBuildError("Copying executables from %s to bin dir %s failed: %s", exedir, targetdir, err)
 
+        # copy data dir
+        datadir = os.path.join(self.cfg['start_dir'], 'data')
+        targetdir = os.path.join(self.installdir, 'data')
+        if os.path.exists(targetdir):
+            self.log.info("Won't copy data dir. Destination directory %s already exists" % targetdir)
+        elif not os.path.exists(datadir):
+            self.log.info("Won't copy data dir. Source directory %s does not exist" % datadir)
+        else:
+            try:
+                shutil.copytree(datadir, targetdir)
+            except:
+                raise EasyBuildError("Copying tests from %s to %s failed", srctests, targetdir)
```

Comment:
@hajgato positive logic is easier to follow, `srctests` doesn't exist, please use `except OSError` and include actual error in error message, so:

```python
if os.path.exists(targetdir):
    self.log.info("Won't copy data dir. Destination directory %s already exists", targetdir)
elif os.path.exists(datadir):
    try:
        shutil.copytree(datadir, targetdir)
    except OSError as err:
        raise EasyBuildError("Copying tests from %s to %s failed: %s", datadir, targetdir, err)
else:
    self.log.info("Won't copy data dir. Source directory %s does not exist", datadir)
```